src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -53,6 +53,9 @@ namespace
             case sf::BlendMode::DstAlpha:         return GL_DST_ALPHA;
             case sf::BlendMode::OneMinusDstAlpha: return GL_ONE_MINUS_DST_ALPHA;
         }
+        assert(false);
+        sf::err() << "Invalid value for sf::BlendMode::Factor! Fallback to GL_ZERO." << std::endl;
```

Comment:
>Susnux is right, because this function is in the anonymous namespace only, so it's not nested in the ```sf``` namespace.

Oops, ok :( I guess I should have checked what namespace the function was in. :P